liquibase-standard/src/main/java/liquibase/snapshot/jvm/SequenceSnapshotGenerator.java

```diff
@@ -266,4 +263,16 @@ protected SqlStatement getSelectSequenceStatement (Schema schema, Database datab
             throw new UnexpectedLiquibaseException("Don't know how to query for sequences on " + database);
         }
     }
+
+    private static RawParameterizedSqlStatement getMSSQLQuery(Schema schema) {
+        String sql = "SELECT SEQUENCE_NAME, START_VALUE, MINIMUM_VALUE AS MIN_VALUE, " +
```

Comment:
Hey Alex, can we keep the usage of StringBuilder here? Otherwise, I think this will result in a CodeQL scan issue because of the concatenated string.